app/Services/MatchingService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace App\Services;
+
+class MatchingService 
+{
+    public function getMatch(array $occupation_1, array $occupation_2) : float
+    {
+        // use occupation_1 to create new array structure: [['label' => 'value'], []...];
+        $occupationOneArray = [];
+        foreach($occupation_1 as $skill){
+            $occupationOneArray[$skill['label']] = $skill['value'];
+        }
```

Comment:
A cleaner way would be to
`$occupationOneArray = array_combine(array_column($occupation_1, 'label'), array_column($occupation_1, 'value'));`